main.ts

```diff
@@ -0,0 +1,85 @@
+interface Data {
+  _id: number;
+  name: string;
+  imageUrl: string;
+  createdAt: string;
+  updatedAt: string;
+  films: string[];
+  shortFilms: string[];
+  tvShows: string[];
+  videoGames: string[];
+  allies: string[];
+  enemies: string[];
+  parkAttractions: string[];
+  sourceUrl: string;
+  url: string;
+}
+
+interface info {
+  data: Data[];
+}
+
+const $row = document.querySelector('.row') as HTMLElement;
+
+async function disneyData(): Promise<void> {
+  try {
+    const response = await fetch('https://api.disneyapi.dev/character');
+    if (!response.ok) {
+      throw new Error(`http error ${response.status}`);
+    }
+    // data got return
+    const data: info = await response.json();
+    console.log('data', data);
+
+    // looping over the character
+    data.data.forEach((charData) => {
+      const $div = document.createElement('div');
+      $div.classList.add('column-fifth');
+
+      const $img = document.createElement('img');
+      $img.src = charData.imageUrl;
+      $img.alt = charData.name;
+
+      // description of character
+      const $p = document.createElement('p');
+      $p.textContent = charData.name;
+
+      $div.appendChild($img);
+      $div.appendChild($p);
+
+      if ($row) {
+        $row.appendChild($div);
+      }
```

Comment:
I would extract this into it's own function so that you can reuse it when you want. Just like how we created a `renderEntry` function